CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+## 15.5.4
+* Fix UMD Build. ([@acdlite](https://github.com/acdlite) in [31e9344](https://github.com/reactjs/prop-types/commit/31e9344ca3233159928da66295da17dad82db1a8))
+* Remove bad package url. ([@ljharb](https://github.com/ljharb) in [158198f](https://github.com/reactjs/prop-types/commit/158198fd6c468a3f6f742e0e355e622b3914048a))
+
+## 15.5.3
+* Remove the dependency on the `react` package. ([@acdlite](https://github.com/acdlite) in [df318bb](https://github.com/reactjs/prop-types/commit/df318bba8a89bc5aadbb0292822cf4ed71d27ace))
+
+## 15.5.2
+* Remove accidental uncompiled ES6 syntax in the published package. ([@acdlite](https://github.com/acdlite) in [e191963](https://github.com/facebook/react/commit/e1919638b39dd65eedd250a8bb649773ca61b6f1))
+
```

Comment:
I think 15.5.2 is the one that actually removed the (implicit) dependency on `react`:

https://unpkg.com/prop-types@15.5.2/index.js

Compare to 15.5.1:

https://unpkg.com/prop-types@15.5.1/index.js